greedy_algorithms/DJnew.cpp

```diff
@@ -0,0 +1,149 @@
+/**
+ * @file
+ * @brief Implementation of Djikstra's Algorithm
+ * @details
+ * Dijkstra's algorithm is an algorithm for finding the shortest paths between nodes in a graph using its neighbors
+ * description containing links, references,
+ * math equations, etc.
+ * @author [Aryan Talekar](https://github.com/aryantalekar20)
+ */
+
+#include <iostream>
+#include <vector>
+#include <set>
```

Comment:
Consider using `uint64_t` for non-negative values (or their appropriate size: `uint32_t`, `uint16_t`, `uint8_t`) or `int64_t` for negative values. Check other parts of the code ([reference](https://en.cppreference.com/w/cpp/types/integer)). **Let us know if you need any help here or in our [Discord server](https://discord.gg/c7MnfGFGa6).** 🙂

Reply:
Thank you! I just joined the discord server. I also enabled GitHub Actions.